packages/fxa-settings/src/components/Avatar/index.tsx

```diff
@@ -0,0 +1,36 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import classNames from 'classnames';
+import { ReactComponent as DefaultAvatar } from './avatar-default.svg';
+
+type AvatarProps = {
+  avatarUrl: string | null;
+  className?: string;
+};
+
+export const Avatar = ({ avatarUrl, className }: AvatarProps) => {
+  if (avatarUrl) {
+    return (
+      <img
+        data-testid="avatar-nondefault"
+        src={avatarUrl}
+        alt="Your avatar"
+        className={classNames('rounded-full', className)}
+      />
```

Comment:
Should we add a background color to this in the event the image fails to load?

Reply:
👍 That'll be nicer than the flash of alt text, although I think we eventually want this to be a small grey spinner similar to what we've got now. I didn't lift that over because I think it can be simplified a bit (and is a bunch of custom SCSS).

Reply:
Works for me. If you want to leave it for now that's fine.